drivers/serial/uart_nrfx_uarte.c

```diff
@@ -549,7 +546,8 @@ static int uarte_instance_init(struct device *dev,
 		CONFIG_UART_##idx##_IRQ_PRI,			\
 		uarte_nrfx_isr,					\
 		DEVICE_GET(uart_nrfx_uarte##idx),		\
-		0)
+		0);						\
+	irq_enable(CONFIG_UART_##idx##_IRQ_NUM)
```

Comment:
I am wondering if this should be UARTE_, instead

Reply:
This is convention from DT and driver names like:
uart_nrfx_uarte.c

Reply:
OK, I see that .fixup defines same macros for UART and UARTE devices